wagtail/test/testapp/migrations/0001_squashed_0073_revisablechildmodel_secret_text.py

```diff
@@ -350,11 +350,13 @@ class Migration(migrations.Migration):
                 ),
                 (
                     "time_from",
-                    models.TimeField(blank=True, null=True, verbose_name="Start time"),
+                    models.TimeField(blank=True, null=True,
```

Comment:
Just a heads up, not sure why but looks like this file contains reformatting of a whole bunch of lines like this that's unrelated to the change being made.

Maybe your editor did this.

I will clean up when merging though.